src/account/sync/input_selection.rs

```diff
@@ -179,9 +228,9 @@ mod tests {
         for _i in 0..20 {
             let mut available_utxos = generate_random_utxos(&mut rng, 25);
             let sum_utxos_picked = sum_random_utxos(&mut rng, &mut available_utxos);
-            let selected = select_input(sum_utxos_picked, available_utxos).unwrap();
+            let selected = select_input(sum_utxos_picked, available_utxos, 127).unwrap();
             assert_eq!(
-                selected.iter().fold(0, |acc, address| { acc + address.balance }),
+                selected.iter().fold(0, |acc, input| { acc + input.output.amount }),
                 sum_utxos_picked
             );
         }
```

Comment:
Wouldn't global const values for 127, 25, 5 and so on be better?

Reply:
That's just for the tests, I don't think we should have global consts for them